tests/unit/test_mypy.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2012-2022 Snowflake Computing Inc. All rights reserved.
+#
+import logging
+import subprocess
+
+def test_mypy_on_module():
```

Comment:
This isn't the right way to enforce `mypy` in the code-base.
Please add it to our `.pre-commit-config,yaml` instead, see https://github.com/pre-commit/mirrors-mypy

Reply:
You'll want to add the other dependencies to `additional_dependencies`

Reply:
will be handled in a separate issue https://snowflakecomputing.atlassian.net/browse/SNOW-705882

Reply:
removed test to address concerns, mypy integration deferred to other issue.